.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
`push` has the same problem as https://github.com/libero/article-store/pull/61?

Reply:
Yes. https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#pull-request-event-pull_request states:
> Note: By default, a workflow only runs when a pull_request's activity type is `opened`, `synchronize`, or `reopened`. To trigger workflows for more activity types, use the types keyword.

This doesn't seem to cover new commits on an open pull request (nor do the other activity types listed there?). Will try.